packages/driver-interface/services/cache.js

```diff
@@ -1,22 +1,16 @@
 const bookingsCache = new Map()
 const vehiclesCache = new Map()
 const instructionsCache = new Map()
+const idMap = new Map()
 
 module.exports = {
   getBooking: (id) => bookingsCache.get(id),
-  removeBooking: (id) => bookingsCache.delete(id),
   addBooking: (id, booking) => bookingsCache.set(id, booking),
-  updateBooking: (id, update) => {
-    const booking = bookingsCache.get(id)
-    bookingsCache.set(id, {
-      ...booking,
-      ...update,
-    })
-  },
   setInstructions: (vehicleId, inscructions) =>
     instructionsCache.set(vehicleId, inscructions),
   getInstructions: (vehicleId) => instructionsCache.get(vehicleId),
-
   addVehicle: (vehicleId, vehicle) => vehiclesCache.set(vehicleId, vehicle),
   getVehicle: (vehicleId) => vehiclesCache.get(vehicleId),
+  getVehicleIdFromTelegramId: (telegramId) => idMap.get(telegramId),
```

Comment:
Am I wrong if I was to say that this should be renamed to "getVehicleId**By**TelegramId", instead of "getVehicleId**From**TelegramId"?

Reply:
Not wrong. I don't think that getVehicleIdFromTelegramId is wrong either, but agreed ById is "more convention"